modules/apps/27-interchain-accounts/types/account_test.go

```diff
@@ -44,27 +45,53 @@ func TestTypesTestSuite(t *testing.T) {
 func (suite *TypesTestSuite) TestGeneratePortID() {
 	var (
 		path  *ibctesting.Path
-		owner string
+		owner = "testing"
 	)
-	var testCases = []struct {
+
+	testCases := []struct {
 		name     string
 		malleate func()
 		expValue string
 		expPass  bool
 	}{
-		{"success", func() {}, "ics-27-0-0-owner123", true},
-		{"success with non matching connection sequences", func() {
-			path.EndpointA.ConnectionID = "connection-1"
-		}, "ics-27-1-0-owner123", true},
-		{"invalid owner address", func() {
-			owner = "    "
-		}, "", false},
-		{"invalid connectionID", func() {
-			path.EndpointA.ConnectionID = "connection"
-		}, "", false},
-		{"invalid counterparty connectionID", func() {
-			path.EndpointB.ConnectionID = "connection"
-		}, "", false},
+		{
+			"success",
+			func() {},
+			"ics-27-0-0-testing",
```

Comment:
Can change `testing` to be a mock address that resembles a bech32 address? Just for the sake of readability for newcomers to ICA.